README.md

```diff
@@ -11,5 +11,29 @@ _bioRxiv_ 2020 doi:10.1101/2020.11.09.355982
 - [`condor`](condor): scripts for running the analyses in a HTCondor cluster
 - [`data`](data): source and target gene lists for the network flow analysis
 
-## TODO
-- update `requirements.txt`
+## Install python dependencies
+```
+# setup virtual environment for flopro dependencies
+python -m venv env
```

Comment:
I'm used to conda instead of venv so I tested with
```
conda create -n flopro python=3.8
conda activate flopro
```

I don't we necessarily need to specify both ways of setting up the environment though.